### ISSUE TITLE
filepath example is failing in language server source gen

### ISSUE THREAD

**Issue by daneshk** [ballerina-platform/ballerina-lang#14821]
**Description:**
Please find the error log below. filepath is a newly introduce module. We currently skip filepath example from SourceGenTest. Please check.

```
[ERROR] formattingSourceGenTests(org.ballerinalang.langserver.sourcegen.SourceGenTest)  Time elapsed: 0.221 s  <<< FAILURE!
java.lang.AssertionError: 
Exception occurred while processing file: filepath_test.bal
Exception:java.lang.NullPointerException
        at org.ballerinalang.langserver.sourcegen.SourceGenTest.formattingSourceGenTests(SourceGenTest.java:89)
Caused by: java.lang.NullPointerException
        at org.ballerinalang.langserver.sourcegen.SourceGenTest.formattingSourceGenTests(SourceGenTest.java:80)

[ERROR] formattingSourceGenTests(org.ballerinalang.langserver.sourcegen.SourceGenTest)  Time elapsed: 0.162 s  <<< FAILURE!
java.lang.AssertionError: 
Exception occurred while processing file: filepath.bal
Exception:java.lang.NullPointerException
        at org.ballerinalang.langserver.sourcegen.SourceGenTest.formattingSourceGenTests(SourceGenTest.java:89)
Caused by: java.lang.NullPointerException
        at org.ballerinalang.langserver.sourcegen.SourceGenTest.formattingSourceGenTests(SourceGenTest.java:80)
```

**Steps to reproduce:**
Remove filepath.bal and filepath_test.bal from ignored list and build language server with test.

**Affected Versions:**
PR #14773

**OS, DB, other environment details and versions:**
N/A

**Comment by daneshk** [comment]
Thanks @NipunaMarcus for finding out the root cause of this issue. 
Root Cause: 
filepath_test.bal file contains below invalid string which cause compilation failure.
```bal
out3 ="Parent of /A/B/C: \A\B";
```
When we correct it like below, It works,
```bal
out3 ="Parent of /A/B/C: \\A\\B";
```

Now the problem is, this ballerina example compiled and ran correctly when we locally. But it should fail.

**Comment by daneshk** [comment]
Change label to compiler to check why it is not failed when we run locally

**Comment by daneshk** [comment]
Closing this issue. Fix is added to #14773
